MvvmCross-Forms/MvvmCross.Forms/Views/MvxFormsPagePresenter.cs

```diff
@@ -172,8 +172,9 @@ public virtual void RegisterAttributeTypes(Dictionary<Type, MvxPresentationAttri
                     if (attribute.WrapInNavigationPage && FormsApplication.MainPage is MvxNavigationPage currentPage)
                     {
                         if (attribute.NoHistory)
```

Comment:
This `if` pattern seems duplicated a lot of places in this class. Can we encapsulate this please?